packages/app-accounts/src/Editor.tsx

```diff
@@ -38,8 +40,16 @@ class Editor extends React.PureComponent<Props, State> {
   }
 
   render () {
+    const { t } = this.props;
+    const { isForgetOpen } = this.state;
     return (
       <div className='accounts--Editor'>
+        <Forgetting
+          isOpen={isForgetOpen}
+          onClose={this.toggleForget}
+          doForget={this.onForget}
+          t={t}
```

Comment:
We don't need to pass t through, just wrap the `Forgetting` component export in a `translate`